services/httpd/handler.go

```diff
@@ -257,23 +257,17 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, user *meta.
 	p := influxql.NewParser(strings.NewReader(qp))
 	db := q.Get("db")
 
+	// Sanitize the request query params so it doesn't show up in the response logger.
+	// Do this before anything else so a parsing error doesn't leak passwords.
+	sanitize(r)
```

Comment:
I don't think we want to run the sanitizer for non-query requests. The response logger doesn't seem to be limited to just `serveQuery`.